org/views.py

```diff
@@ -143,4 +143,41 @@ def Permission_Set_List(request,org_id):
             }
             response_object.append(value)
         return Response(response_object, status.HTTP_200_OK)
-    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
\ No newline at end of file
+    return Response({"detail": "You are not authorised to view this."}, status.HTTP_403_FORBIDDEN)
+
+
+
+@swagger_auto_schema(
+    operation_id="edit_org",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='PUT',
+    request_body = EditOrgSerializer,
+    responses={
+        '200': set_example(responses.update_org_200),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '403': set_example(responses.admin_access_403),
+        '500': set_example(responses.empty_fields_500)
+    }
+)
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def EditOrg(request,org_id):
```

Comment:
Also, always follow some standard rules like:
Naming of function and variables is lowercase, underscores used for spacing.
Name of Classes in uppercase.

I guess we need to mark the linting issue as High priority! @naveennvrgup

Reply:
Still some basic things should be taken care by ourselves, rather than depending on linter.
Refer this : https://www.python.org/dev/peps/pep-0008/

Reply:
Actually previous functions are also CamelCase so I kept it like that, ```org_id``` was also used before by @anshsrtv so I guess that's fine too, anyways I'll look that up in future and improve.

Reply:
> I guess we need to mark the linting issue as High priority! @naveennvrgup

I totally agree with @BadduCoder . I am changing the label of #95 .

Reply:
Once we set up a linting hook in git these issues will be solved. So I am resolving this conversation for now.